util/grpc/src/grpcio_extensions.rs

```diff
@@ -61,6 +61,15 @@ pub trait ConnectionUriGrpcioServer {
     /// hot-reloading certificates when TLS is used.
     #[must_use]
     fn bind_using_uri(self, uri: &impl ConnectionUri, logger: Logger) -> Self;
+
```

Comment:
Optional: IIUC we can add a helper method that calls `channel_args`, e.g.

```suggestion

/// Set the channel args to our defaults.
fn set_default_channel_args(self, env: Arc<Environment>) -> Self {
  self.channel_args(Self::default_channel_builder(env).build_args())
}
```

Reply:
good idea, added.